devel/gk/Portfile

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+
+name                    gk
```

Comment:
`github.setup` sets `name` and `version` for you.

You can move `githup.setup` up here, and delete `name` and `version`.  You will also have to import the `github` portgroup right after the `PortSystem` line.

```
PortSystem              1.0
PortGroup               github 1.0

github.setup            gitkraken gk-cli 1.0.6 v
github.tarball_from     releases
...
```

Reply:
I agree, but then the the port should be either put in a directory called "gk-cli" or the name should be set after the github.setup line to "gk".

Reply:
The binary is named `gk` but the project in GitHub is called `gk-cli`. I don't mind changing the name of the directory as long as we keep the binary name. Thanks for the feedback!

Reply:
the binary name will not change, that remains what upstream uses. However, the portname and directory should match, hence my comment. 

If you want to be able to do "sudo port install gk", then add a "name" field after the "github.setup" line, otherwise change the directory name.

Reply:
I'll add the name attr so we can keep the `gk` name if that's ok.